### PR TITLE
Remove some dead assignments in L1Trigger/DTTriggerPhase2/src/HoughGrouping.cc

### PR DESCRIPTION
**Pull request by perrotta**
#### PR description:

Title says all.
Spotted by the static analyzer
It should not interfere with any of the ongoing L1T related developments, but if so this PR can be simply postponed

#### PR validation:

It builds.

### CODE REVIEW

**Comment by perrotta** [comment]
please test

**Comment by cecilecaillol** [comment]
+l1

File: L1Trigger/DTTriggerPhase2/src/HoughGrouping.cc [hunk 1]
```diff
@@ -683,18 +679,12 @@ ProtoCand HoughGrouping::associateHits(const DTChamber* thechamb, double m, doub
...
           LocalPoint tmpLocalCh_r = thechamb->toLocal(tmpGlobal_r);
 
-          distleft = std::abs(thepoint.x() - tmpLocalCh_l.x());
-          distright = std::abs(thepoint.x() - tmpLocalCh_r.x());
-
           // Filling info
           returnPC.nLayersWithHits_++;
           returnPC.isThereNeighBourHitInLayer_[abslay] = true;
 
-          returnPC.xDistToPattern_[abslay] = abs(tmpx - (tmpLocalCh.x() - 1.05));
```
Comment (srimanob) [inline L1Trigger/DTTriggerPhase2/src/HoughGrouping.cc:693]: Don't we need these three returnPC(s) at all? Were it there on purpose?

Comment (perrotta) [reply L1Trigger/DTTriggerPhase2/src/HoughGrouping.cc:693]: They were overwritten two lines below...

Comment (srimanob) [reply L1Trigger/DTTriggerPhase2/src/HoughGrouping.cc:693]: Ah, OK. Thanks. 
There is `else` of `if (isDistRight)` which did not show up on my screen.

**Comment by srimanob** [comment]
+Upgrade

Technical PR, no change is expected.

**Comment by perrotta** [comment]
+1

**Pull request merged**